app/bundles/EmailBundle/Tests/Controller/EmailControllerTest.php

```diff
@@ -141,7 +142,7 @@ protected function setUp(): void
             $this->corePermissionsMock
         );
         $this->controller->setContainer($this->containerMock);
-        $this->sessionMock->method('getFlashBag')->willReturn($this->flashBagMock);
+        $this->sessionMock->method('getFlashBag')->willReturn($this->createMock(FlashBagInterface::class));
```

Comment:
I think this should be reverted, because if one would test something with the flashes, should have the same instance in the constructor and session.

Reply:
Same as the above comment.